src/hooks/useUserByNameOrAddress.ts

```diff
@@ -8,6 +8,7 @@ const useUserByNameOrAddress = (usernameOrAddress: string) => {
       name: usernameOrAddress,
       address: usernameOrAddress,
     },
+    fetchPolicy: 'cache-and-network',
```

Comment:
This ensures the avatar is uptodate when you change it in "Settings" and then navigate to "My Profile"